mmdet/models/detectors/yolact.py

```diff
@@ -0,0 +1,113 @@
+import torch
+
+from mmdet.core import bbox2result, multi_apply
+from .. import builder
+from ..registry import DETECTORS
+from .single_stage import SingleStageDetector
+
+
+@DETECTORS.register_module
+class Yolact(SingleStageDetector):
+
+    def __init__(self,
+                 backbone,
+                 neck,
+                 bbox_head,
+                 segm_head,
+                 mask_head,
+                 train_cfg=None,
+                 test_cfg=None,
+                 pretrained=None):
+        super(Yolact, self).__init__(backbone, neck, bbox_head, train_cfg,
+                                     test_cfg, pretrained)
+        self.segm_head = builder.build_head(segm_head)
+        self.mask_head = builder.build_head(mask_head)
+        self._init_weights()
+
+    def _init_weights(self, pretrained=None):
+        self.segm_head.init_weights()
+        self.mask_head.init_weights()
+
+    def process_gt_single(self, gt_bboxes, gt_labels, gt_masks, device):
+        """Cuda the gt_masks and discard boxes that are smaller than we'd like
+        """
+        gt_masks = torch.from_numpy(gt_masks).to(device)
+        w = gt_bboxes[:, 2] - gt_bboxes[:, 0]
+        h = gt_bboxes[:, 3] - gt_bboxes[:, 1]
+        keep = (w > 4.0) * (h > 4.0)
```

Comment:
Make 4.0 configurable. For example add parameters `min_box_width` and `min_box_height`

Reply:
Done. It's called `min_box_wh` in the `train_cfg`.